docs/help/Extended-API-Guideline.md

```diff
@@ -0,0 +1,63 @@
+Extended API Guideline
+======================
+
+Basically, our basic APIs are based on node.js. They will follow same form with node.js because of compatibility.<br>
+However, extended APIs need a guideline because they are implemented by many contributor. (for consistent usability)
+
+
+# Ground Rules
+
+## API naming rules
+1. The APIs which have similar role should have same API name. 
+2. Basically, all APIs are async API. If you want to make sync API, you need to add ``Sync`` as a suffix.<br>For example, ``readSync()``, ``writeSync()``, and so on.
```

Comment:
Single quotes (eg.: \`Sync\`) should be used for inline code snippets and triple quotes for multiline code snippets.